translator/conventions/opentelemetry.go

```diff
@@ -0,0 +1,101 @@
+// limitations under the License.
+
+package conventions
+
+// OpenTelemetry Semantic Convention values for Resource attribute names.
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+const (
+	AttributeServiceName      = "service.name"
+	AttributeServiceNamespace = "service.namespace"
+	AttributeServiceInstance  = "service.instance.id"
+	AttributeServiceVersion   = "service.version" // proposed
+	AttributeLibraryName      = "library.name"
+	AttributeLibraryLanguage  = "library.language"
+	AttributeLibraryVersion   = "library.version"
+	AttributeContainerName    = "container.name"
+	AttributeContainerImage   = "container.image.name"
+	AttributeContainerTag     = "container.image.tag"
+	AttributeK8sCluster       = "k8s.cluster.name"
+	AttributeK8sNamespace     = "k8s.namespace.name"
+	AttributeK8sPod           = "k8s.pod.name"
+	AttributeK8sDeployment    = "k8s.deployment.name"
+	AttributeHostHostname     = "host.hostname"
+	AttributeHostID           = "host.id"
+	AttributeHostName         = "host.name"
+	AttributeHostType         = "host.type"
+	AttributeCloudProvider    = "cloud.provider"
+	AttributeCloudAccount     = "cloud.account.id"
+	AttributeCloudRegion      = "cloud.region"
+	AttributeCloudZone        = "cloud.zone"
+)
+
+// OpenTelemetry Semantic Convention values for general Span attribute names.
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md
+// Peer attributes currently defined in database conventions
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-database.md
+const (
+	AttributeComponent   = "component"
+	AttributePeerAddress = "peer.address"  // to be obsoleted
+	AttributePeerHost    = "peer.hostname" // to be obsoleted
+	AttributePeerIpv4    = "peer.ipv4"     // to be obsoleted
+	AttributePeerIpv6    = "peer.ipv6"     // to be obsoleted
+	AttributePeerPort    = "peer.port"     // to be obsoleted
+	AttributePeerService = "peer.service"  // to be obsoleted
+)
+
+// OpenTelemetry Semantic Convention values for component attribute values.
+// Possibly being removed due to issue #336
+const (
+	ComponentTypeHTTP = "http"
+	ComponentTypeGRPC = "grpc"
+)
+
+// OpenTelemetry Semantic Convention attribute names for HTTP related attributes
+// See: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md
+const (
+	AttributeHTTPMethod     = "http.method"
+	AttributeHTTPURL        = "http.url"
+	AttributeHTTPTarget     = "http.target"
+	AttributeHTTPHost       = "http.host"
+	AttributeHTTPScheme     = "http.scheme"
+	AttributeHTTPStatusCode = "http.status_code"
+	AttributeHTTPStatusText = "http.status_text"
+	AttributeHTTPFlavor     = "http.flavor"
+	AttributeHTTPServerName = "http.server_name"
+	AttributeHTTPPort       = "http.port"
```

Comment:
[spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#http-server-semantic-conventions) has host.port. I think spec needs to change to http.port

Reply:
Submitted spec issue: https://github.com/open-telemetry/opentelemetry-specification/issues/369